ci/jenkins/README.md

```diff
@@ -93,60 +93,62 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
```

Comment:
line nos 88 - 93 are related to e2e tests running on docker runtime, you can modify it for containerd and retain below part for conformance, then no need to add lines 104-108